doc/radosgw/multisite.rst

```diff
@@ -0,0 +1,487 @@
+- **Zonegroup**: A zonegroup consists of multiple zones, this approximately
+  corresponds to what used to be called as a region in pre Jewel releases for
+  federated deployments. There should be a master zonegroup that will handle
+  changes to the system configuration.
+
+- **Zonegroup map**: A zonegroup map is a configuration structure that holds the
+  map of the entire system, ie. which zonegroup is the master, relationships
+  between different zonegroups and certain configurables like storage policies.
+
+- **Realm**: A realm is a container for zonegroups, this allows for separation
+  of zonegroups themselves between clusters. It is possible to create multiple
+  realm, making it easier to run completely different configurations in the same
+  cluster.
+
+- **Period**: A period holds the configuration structure for the current state
+  of the realm, a period needs to be updated whenever there is a zone
+  configuration change of the master zone.
+
+
+
+Pool Configuration
+==================
+
+About this guide
+================
+
+In this guide we create a single zone group, with two seperate zones, which
+actively sync data between them. There is no sync agent involved for mirroring
+data changes between the RadosGWs and this allows for a much simpler
+configuration scheme and active-active configurations. Please note that metadata
+operations such as creating a user would still need to go through the master
+zone, however data operations such as creation of buckets objects etc. can be
+handled by any of the zones.
+
+System Keys
+===========
+
+While configuring zones, RadosGW, expects creation of zone a system user, with
+an S3 like access and secret keys. This allows another radosgw instance to pull
+the configuration remotely, with the access and secret keys. For making
+scripting and use of configuration management tools easier, it would be easier
+to generate the access-key and secret-key before hand and use these. For the
+purpose of the guide we assume an access key and secret key are set in the
+environment. For eg., something like::
+
+  $ SYSTEM_ACCESS_KEY=1555b35654ad1656d805
+  $ SYSTEM_SECRET_KEY=h7GhxuBLTrlhVUyxSPUKUV8r/2EI4ngqJxD7iBdBYLhwluN30JaT3Q==
+
+Naming conventions
+==================
+
+This section describes the process of setting up a master zone. For the purpose
+of the guide, we'll assume a zonegroup called ``us`` spanning the United States,
+which will be our master zonegroup. This will contain two zones written in a
+``{zonegroup}-{zone}`` format, this is just a convention only and you are free
+to choose a format you prefer to set these names. So in summary:
+
+- Master Zonegroup: United States ``us``
+- Master zone: United States, East Region: ``us-east``
+- Secondary zone: United States, West Region: ``us-west``
+
+This will be a part of a larger realm, say ``gold``
+
+Creating a realm
+----------------
+
+Configure a realm called ``gold``, which for ::
+
+  # radosgw-admin realm create --rgw-realm=gold
```

Comment:
I would add the --default flag to set the realm as default so you don't need to provide in all the admin commands